drivers/flash/flash_stm32l5_u5.c

```diff
@@ -0,0 +1,405 @@
+/*
```

Comment:
I wonder why this is appearing as a new file introduction.
Maybe you need to remove the L5 one ?